webpush/realpush_test.go

```diff
@@ -0,0 +1,51 @@
+package webpush
```

Comment:
I started working on selenium tests, but the  libraries are sketchy at best in terms of their implementation and one key feature used in Firefox and Chrome is the ability to set profiles / preferences. There is a Firefox Profile Struct but it has an unusual constructor, asked on the project for clarification, but not hopeful. Will carry on exploring for Go.